pkgs/by-name/tu/turnon/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  cairo,
+  pango,
+  pkg-config,
+  libadwaita,
+  blueprint-compiler,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "turnon";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "swsnr";
+    repo = "turnon";
+    rev = "v${version}";
+    hash = "sha256-TF6mGETX6cK0ob2LLrjneifwSf+9EEjrv4x7AfWIamQ=";
+  };
+
+  cargoHash = "sha256-EpwgvaunShETZqH7v3Dux60pGLeBOtBZhpbDi7Eows4=";
+
+  nativeBuildInputs = [
+    cairo
+    pango
+    pkg-config
+    blueprint-compiler
```

Comment:
```suggestion
    blueprint-compiler
    wrapGAppsHook4
```

Reply:
Hi.
I had `wrapGAppsHook4` in the nativeBuildInputs but then I removed it and it still worked. I wasn't sure if it was necessary.

Reply:
Removing it may produce UI or functionality breakage in special cases, such as icons in file selection dialogs falling back to adwaita. I recommend you add this to all GTK3/4 applications.

Reply:
got it 👍